style.css

```diff
@@ -47,47 +51,123 @@ header {
   position: relative;
 }
 
+
 .universe-background {
   position: absolute;
   width: auto;
-  margin-top: 0;
+  top: -10%;
   z-index: 0;
+  left: -55%;
 }
 
 .little-prince {
   width: 60%;
   z-index: 1;
+  position: absolute;
+  padding-top: 30%;
+  padding-left: 35%;
+}
+
+
+
+.sidenav {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: #9591F2;
+  overflow-x: hidden;
+  transition: 0.5s;
+  padding-top: 60px;
+}
+
+.sidenav a {
+  padding: 8px 8px 8px 32px;
+  text-decoration: none;
+  display: block;
+  transition: 0.3s;
+}
+
+.overview-title {
+  font-family: GoodDog New;
```

Comment:
Same deal here, double check these fonts are actually loaded! I don't see them in your code right now. Also, consider providing a generic fallback family, like `sans-serif`.